deploy/handler.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
```

Comment:
please try running this with openshift. i think we will need to add scc there

Reply:
So check-patch.os-3.11.0.el7.x86_64 is passing so we are all good ?

Reply:
Not sure, it may get created, but failt to schelute pods. Please trz it locally

Reply:
confirmed that it fails 
        Warning   FailedCreate                      daemonset-controller                 Error creating: pods "nmstate-handler-" is forbidden: unable to validate against any security context constraint: [spec.volumes[0]: Invalid value: "hostPath": hostPath volumes are not allowed to be used spec.containers[0].securityContext.privileged: Invalid value: true: Privileged containers are not allowed]
1s        33s       3         ansible-service-broker.15a725c5e9548158            ClusterServiceBroker

Reply:
we need to add some SCC config

Reply:
Done